curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/64646db2c684b7a3a174a1d0.md

```diff
@@ -7,15 +7,15 @@ dashedName: step-81
 
 # --description--
 
-If you test your code, you'll notice that your console logs are not in the expected order. Instead of logging `free`, pausing for a second before logging `Code`, and finally logging `Camp`, you'll see this:
+If you test your code, you'll notice that your console logs are not in the expected order. Instead of logging `"free"`, pausing for a second before logging `"Code"`, and finally logging `"Camp"`, you'll see this:
 
 ```md
 free
 Camp
 Code
 ```
 
-This is because the `setTimeout()` function is <dfn>asynchronous</dfn>, meaning that it doesn't stop the execution of the rest of your code. All the code in the `showAnimation()` function runs line by line, but because `setTimeout()` is asynchronous, `free` and `Camp` are logged to the console immediately, and then `Code` is logged to the console after a one second delay.
+This is because the `setTimeout()` function is <dfn>asynchronous</dfn>, meaning that it doesn't stop the execution of the rest of your code. All the code in the `showAnimation()` function runs line by line, but because `setTimeout()` is asynchronous, `"free"` and `"Camp"` are logged to the console immediately, and then `"Code"` is logged to the console after a one second delay.
```

Comment:
in this case here, what is being logged to the console will be `free`, `camp` and `code` without the `" "`.